navigation2/README.md

```diff
@@ -37,13 +38,28 @@ There is a run.sh script provided for convenience that will run the spaceros ima
 ./run.sh
 ```
 
-Upon startup, the container automatically runs the entrypoint.sh script, which sources the MoveIt2 and Space ROS environment files.
+Upon startup, the container automatically runs the entrypoint.sh script, which sources the Nav2 and Space ROS environment files.
 You'll now be running inside the container and should see a prompt similar to this:
 
 ```
-root@ip-your-ip-address:/home/spaceros-user/nav2_ws#
+spaceros-user@ip-your-ip-address:/home/spaceros-user/nav2_ws#
 ```
 
 ## Running Navigation2 Demo
 
-To run the latest demo, see the README in the nav2_demos folder
+To run the latest demo, see the README in the [nav2_demo](../nav2_demo/README.md) folder.
+
+
+## Updating navigation2 packages
+The `navigation2.repos` file available in this repository provides a list of repos that are required to build the nav2 stack. ROS packages already included in the SpaceROS base image are omited from this file.
+
+To update navigation2 packages used to build the space nav2 image, run the `docker_update_nav2_repos.sh` script:
+
+```
+./docker_update_nav2_repos.sh
```

Comment:
I am trying to run this script. i have a local osrf/space-ros:latest that I just built from space-ros main branch (including PR 170).

but i get an error:

```
~/git/space-ros/docker/navigation2 (pr152_pin_nav2)$ ./docker_update_nav2_repos.sh 
Error: Either packages or repos (or both) have to be specified
Generate .repos file with list of SpaceROS repositories to clone
Usage: ./generate_repos.sh --packages PACKAGES --outfile OUTFILE [--excluded-packages EXCLUDED_PACKAGES] [--repos REPOS] [--rosdistro ROSDISTRO] [--upstream UPSTREAM] [--exclude-installed EXCLUDE_INSTALLED]
  --outfile                   The output file
  --packages                  File containing a list of ROS packages to include
  --repos                     File containing a list of ROS repos to use.
                              Use as alternative to 'packages', rosinstall can generate .repos file from list of repos instead of list of packages
  --excluded-packages         File containing a list of packages to exclude [optional] 
  --rosdistro                 ROS2 distribution (default: humble)
  --upstream                  Specify whether to use version tags of upstream repositories (default: true)
  --exclude-installed         Whether to exclude already installed packages from .repos file. Installed workspaces must be sourced to make it work (default: true)
  -h, --help                  Display this help and exit
```

Reply:
ah, $ROS_DISTRO vs $ROSDISTRO in `update_nav2_repos.sh`

Reply:
Ok, so it seems that I need to fix it. Will run it again and check if it works. I plan to solve it today/tomorrow

Reply:
Fixed